docs/pages/access-controls/guides/moderated-sessions.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: Moderated Sessions
+description: Session moderation with RBAC and humans
+h1: Moderated Sessions
+---
+
+## Introduction
+
+Moderated sessions allows Teleport administrators to
+define requirements for other users to be present in a Server or Kubernetes Access session.
+
+Moderated Sessions are useful in the following scenarios:
+- When you have stringent security requirements and need to have people watching over user-initiated sessions to a set of servers.
+- When you want to share a terminal with someone else to be able to instruct or collaborate.
+
+## Core Features
+
+With Moderated Sessions you can:
+- Join SSH and Kubernetes sessions from the terminal as an observer, moderator or peer.
+- Require auditors to be present in certain sessions.
+- Allow people to join your session to observe what you are doing in real time.
+
+## Getting Started
+
+In this guide we'll be using Moderated Sessions to define required participants for a Kubernetes session.
+This guide assumes an already running Teleport cluster with a connected Kubernetes cluster named `minikube`.
+
+<Admonition type="tip">
+  If you don't have a cluster that meets the aforementioned requirements,
+  you can create one by following [this guide](https://goteleport.com/docs/kubernetes-access/getting-started/).
+</Admonition>
+
+### Step 1/4: Create the prod-access role
+
+```code
```

Comment:
This snippet is going to look a bit odd until [this issue](https://github.com/gravitational/next/issues/811) is fixed, but I don't think we should block on that issue.

Reply:
Agreed, not much we can do here except continue